pytorch_finufft/functional.py

```diff
@@ -696,8 +696,12 @@ def get_nufft_func(
     dim: int, nufft_type: int, device_type: str
 ) -> Callable[..., torch.Tensor]:
     if device_type == "cuda":
```

Comment:
not actually related to this PR and more of a question: Is `device_type` something that is particular to our code base or is it a pytorch concept? If it is not a pytorch concept, we could consider receiving here a pytorch device, which can be a device object or a string indicating the device, which can be regularized by calling `torch.device` on it.

Reply:
Yes, this string comes from calling `.type` on a `torch.device`.

The advantage over a device string is that you don’t have to deal separately with “cuda”, “cuda:0”, etc